staking/programs/integrity-pool/src/state/pool.rs

```diff
@@ -253,7 +256,7 @@ impl PoolData {
             i += 1;
         }
 
-        for j in 0..(publisher_caps.num_publishers() as usize) {
+        for j in 0..publisher_caps.num_publishers() as usize {
```

Comment:
weirdly the u16 to usize conversion error can't be converted to AnchorError using the ? so I'm not updating this